javascript/ql/src/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -1017,6 +1020,10 @@ module NodeJSLib {
     }
   }
 
+  private class ClientRequestEventEmitter extends NodeJSEventEmitter {
+    ClientRequestEventEmitter() { this = any(ClientRequestHandler handler).getAParameter() }
+  }
```

Comment:
I prefer some evaluation of what new results we get here. 
(Possibly post-sprint).

Reply:
LGTM.com run in progress: https://lgtm.com/query/180444781372489199/ (without the shorthand request below)

Reply:
oh, that doesn't look good. Let me fix.

Reply:
Fixed. I also found that `http.request(...)` returns an event emitter, I have snuck it in here since we need to evaluate it thoroughly later.